tests/fermi/test_fermionic.py

```diff
@@ -125,6 +125,20 @@ def test_init_error(self, operator):
         with pytest.raises(ValueError, match="The operator indices must belong to the set"):
             FermiWord(operator)
 
+    def test_to_mat(self):
+        """Test that the matrix representation of FermiWord is correct."""
+        fw = FermiWord({(0, 0): "+", (1, 1): "-"})
+        expected_mat = np.zeros((4, 4), dtype=complex)
+        expected_mat[2, 1] = 1.0
+
+        mat = fw.to_mat()
+        assert np.allclose(mat, expected_mat)
+
+        # check that the error is raised when the request matrix dimension is smaller
+        # than that determined by the largest index
+        with pytest.raises(ValueError, match="n_orbitals cannot be smaller than 2"):
```

Comment:
Please separate the ValueError tests into new one(s).

Reply:
Done!